src/libraries/System.Numerics.Tensors/tests/TensorPrimitivesTests.netcore.cs

```diff
@@ -35,6 +36,29 @@ public static void ConvertToHalf(int tensorLength)
             }
         }
 
+        [Theory]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/92885", typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser), nameof(PlatformDetection.IsMonoAOT))]
```

Comment:
```suggestion
```
This was fixed in 'release/8.0' by https://github.com/dotnet/runtime/pull/93677

Reply:
I was just waiting for that PR to merge so I could take these changes. Thanks for the heads up, I'll get those in now.